Subject: [PATCH] [FEAT VS1804] zipl/boot: fix comment in stage3.lds
From: Marc Hartmayer <mhartmay@linux.ibm.com>

Summary:     genprotimg: Introduce new tool for the creation of PV images
Description: genprotimg takes a kernel, host-key documents, optionally an
             initrd, optionally a file with the kernel command line, and it
             generates a single, loadable image file. The image consists of a
             concatenation of a plain text boot loader, the encrypted
             components for kernel, initrd, and cmdline, and the
             integrity-protected PV header, containing metadata necessary for
             running the guest in PV mode. It's possible to use this image file
             as a kernel for zIPL or for a direct kernel boot using QEMU.
Upstream-ID: 971970989bbb1de8887d11b0ab8e4f19adbd484f
Problem-ID:  VS1804

Upstream-Description:

             zipl/boot: fix comment in stage3.lds

             See STAGE3_STACK_ADDRESS macro.

             Reviewed-by: Jan Höppner <hoeppner@linux.ibm.com>
             Signed-off-by: Marc Hartmayer <mhartmay@linux.ibm.com>
             Signed-off-by: Jan Höppner <hoeppner@linux.ibm.com>


Signed-off-by: Marc Hartmayer <mhartmay@linux.ibm.com>
---
 zipl/boot/stage3.lds |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/zipl/boot/stage3.lds
+++ b/zipl/boot/stage3.lds
@@ -10,7 +10,7 @@
  * 0x6000-0x8fff	free
  * 0x9000-0x9fff        Stage3 parameter
  * 0xa000-0xdfff        Stage3 code + data
- * 0xe000-0xffff        Stack
+ * 0xf000-0xffff        Stack
  */
 
 SECTIONS
